### PR TITLE
feat(as/configuration): add more parameters and support importing

### PR DESCRIPTION
**Pull request by ShiChangkuo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

do the following changes on `huaweicloud_as_configuration`:

- support importing
- add parameters in `instance_config`: **security_group_ids**, **flavor_priority_policy**, **ecs_group_id**, **charging_mode**
- add `status` attribute

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ make testacc TEST='./huaweicloud/services/acceptance/as' TESTARGS='-run=TestAccASConfiguration'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/as -v -run=TestAccASConfiguration -timeout 360m -parallel 4
=== RUN   TestAccASConfiguration_basic
=== PAUSE TestAccASConfiguration_basic
=== RUN   TestAccASConfiguration_instance
=== PAUSE TestAccASConfiguration_instance
=== CONT  TestAccASConfiguration_basic
=== CONT  TestAccASConfiguration_instance
--- PASS: TestAccASConfiguration_basic (31.44s)
--- PASS: TestAccASConfiguration_instance (158.67s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/as        158.734s
```

### CODE REVIEW

File: docs/resources/as_configuration.md [hunk 1]
```diff
@@ -195,3 +231,31 @@ The `personality` block supports:
...
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID in UUID format.
+* `status` - The AS configuration status, the value can be **Bound** or **Unbound**.
+
+## Import
+
+AS configuration can be imported by their `id`, e.g.
```
Comment (Lance52259) [inline docs/resources/as_configuration.md]:
```suggestion
AS configurations can be imported by their `id`, e.g.
```

File: huaweicloud/services/acceptance/as/resource_huaweicloud_as_configuration_test.go [hunk 2]
```diff
@@ -144,14 +165,35 @@ func testAccASConfiguration_basic(rName string) string {
...
-    image  = data.huaweicloud_images_image.test.id
-    flavor = data.huaweicloud_compute_flavors.test.ids[0]
+    image              = data.huaweicloud_images_image.test.id
+    flavor             = data.huaweicloud_compute_flavors.test.ids[0]
+    key_name           = huaweicloud_compute_keypair.acc_key.id
+    security_group_ids = [data.huaweicloud_networking_secgroup.test.id]
+
+    metadata = {
+    some_key = "some_value"
+    }
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/as/resource_huaweicloud_as_configuration_test.go]:
```suggestion
    metadata = {
      some_key = "some_value"
    }
```

Comment (ShiChangkuo) [reply huaweicloud/services/acceptance/as/resource_huaweicloud_as_configuration_test.go]: fixed

File: docs/resources/as_configuration.md (hunk 1 above)
Comment (ShiChangkuo) [reply docs/resources/as_configuration.md]: fixed

**Pull request merged**